dss-api.yml

```diff
@@ -126,6 +126,73 @@ definitions:
       - uuid
       - name
       - versions
+  blob_common_metadata:
+    type: object
+    description: Common fields between the files list of a bundle and the metadata tracking an unique file.
+    properties:
+      uuid:
+        type: string
+        description: File unique ID
+      timestamp:
+        type: integer
+        format: int64
+        description: Timestamp of file creation measured in milliseconds since epoch
```

Comment:
For documentation purposes we may want to say "RFC 3339 compliant timestamp string"